test/drivers/test_suites/syscall_exit_suite/recvmsg_x.cpp

```diff
@@ -571,12 +571,17 @@ TEST(SyscallExit, recvmsg_ancillary_data)
 	{
 		/* Create a socket. It is used to pass it to the child process, just for test purposes */
 		int sock = socket(AF_UNIX, SOCK_STREAM, 0);
-		msg = {
+		/*msg = {
```

Comment:
We can cleanup this commented code i guess!

Reply:
Of course, but this part doesn't seem to need to be modified, haha.